cql2/standard/requirements/accent-case-insensitive-comparison/REQ_accenti-builtin-function.adoc

```diff
@@ -0,0 +1,11 @@
+[[req_accent-case-insensitive-comparison_accenti-builtin-function]]
+[width="90%",cols="2,6a"]
+|===
+^|*Requirement {counter:req-id}* |*/req/accent-case-insensitive-comparison/accenti-builtin-function*
+^|A |The server SHALL support a built-in function named `ACCENTI`.
+^|B |The function SHALL accept one argument that can be a character string literal, the name of a property that evaluates to a string literal or a function that returns a string literal (see rules `characterLiteral`, `propertyName`, `function`).
+^|C |The function SHALL return a character string literal.
+^|D |The function SHALL implement https://www.w3.org/TR/charmod-norm/#unicodeNormalization[unicode normalization] described in the implementation guidelines of https://www.unicode.org/versions/Unicode14.0.0[The Unicode Standard, Version 14.0] (see https://www.unicode.org/versions/Unicode14.0.0/ch05.pdf[clause 5.6 Normalization]).
```

Comment:
Consider adding the term “Unicode normalization” to the terms & definitions.

 **Unicode normalization**
normalization
process of removing alternate representations of equivalent sequences from textual data, to convert the data into a form that can be binary-compared for equivalence

Source: https://www.unicode.org/glossary/#normalization

Reply:
Meeting 2022-01-17: Agreed.